adoc/Private-Registry-Airgap.adoc

```diff
@@ -0,0 +1,271 @@
+== Deployment in an Air-Gapped Environment 
+
+=== Introduction
+
+K3s is a lightweight Kubernetes distribution from Rancher that can be
+deployed on a single node. 
+
+=== Use Cases
+
+Testing:: The customer wants to test {sprpbh}, but they do not
+have a Kubernetes cluster prepared.
+Single-node setup:: A production setup for customers that want to use {sprpbh},
+but do not want to deploy CaaSP, Racher's RKE or any other Kubernetes cluster.
+Air-gapped environments:: K3s can be deployed without access to the Internet.
+With {sprpbh} running on such a cluster, it could for example serve as an image
+registry for other Kubernetes clusters in the customer's network.
+
+=== Requirements
+
+You should have Helm installed on the machine where you want to install k3s.
+
+Make sure that you have enough disk space. Rancher does not mention any specific
+requirements for disk size, so prepare at least a few gigabytes. It is possible
+to put the `/var/lib/rancher` directory on any partition or disk where you know
+that you have enough space. In case space starts to run out, you might encounter
+https://github.com/rancher/k3s/issues/1552[k3s issue #1552].
+
+Also check for the latest available _stable_ version of k3s. We do no recommend
+running pre-release versions. At the time of writing, the current stable version
+was `v1.18.13+k3s1`, but this will probably have changed by the time you read
+this. If you encounter 404 errors trying to download packages from GitHub, this
+is probably due to the release of a newer version.
+
+=== Deployment
+
+. We recommend that you read
+https://rancher.com/docs/k3s/latest/en/installation/airgap/#manually-deploy-images-method[the upstream guide]
+on deploying k3s into an air-gapped environment.
+
+. Before you begin, download all the packages for the installation.
+
+.. You will need to download three different files to install k3s.
+
+... The k3s binary from: https://github.com/rancher/k3s/releases
+
+... The k3s install script, which you can, for example, fetch with:
++
+[source,bash]
+----
+curl https://get.k3s.io -o install.sh
+----
+
+... The bundle of images required for basic k3s functionality and for some
+optional features (such as the Traefik ingress controller), from here:
+https://github.com/rancher/k3s/releases/download/v1.18.13%2Bk3s1/k3s-airgap-images-amd64.tar[k3s-airgap-images-amd64.tar]
+
+.. Fetch the images required for the SUSE Private Registry:
++
+[source,bash]
+----
+mkdir images
+for component in core nginx notary-server notary-signer trivy-adapter registryctl jobservice registry db redis; do
+ image="registry.suse.com/harbor/harbor-$component:2.1.0"; \
+ docker pull $image; docker save -o "images/harbor-$component-2.1.0.tar" $image; \
```

Comment:
Please replace `2.1.0` with `2.1.1`

Reply:
or maybe do :2.1, which will automatically update?

Reply:
I have changed this to use 2.1.2 but I have no way to test this, so please verify that I have got it right.

Reply:
I also note that what looks to me like a version number is embedded here:

`export HELM_EXPERIMENTAL_OCI=1`
`helm chart pull registry.suse.com/harbor/harbor:1.5`
`helm chart export registry.suse.com/harbor/harbor:1.5`

Does that need to be 2.1.2 as well, @jsuchome ?

Reply:
Nope, 1.5 is correct here. Version of the helm chart is different than the versions of images